app/views/crn-enter.njk

```diff
@@ -13,7 +13,67 @@
 {% block content %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+    <form method="POST" autocomplete="off" novalidate>
+    <input type="hidden" name="crumb" value="{{crumb}}"/>
     <h1 class="govuk-heading-l">Enter your customer reference number (CRN)</h1>
+    <p class="govuk-body">Enter your CRN number so we can identify you. This is the 10 digit number emailed to you when you created your Rural Payments account.</p>
+
+    {{ 
+      govukInput({
+        label: {
+          text: "Enter your CRN number",
+          classes: "govuk-label",
+          for: "crn"
+        },
+        classes: "govuk-input govuk-input--width-10",
+        id: "crn",
+        name: "crn",
+        value: crn,
+        errorMessage:  { 
+          text: errors.crnErrorMessage
+        }
+      }) 
+    }}
+
+    {{ 
+      govukInput({
+        label: {
+          text: "Confirm your CRN number",
+          classes: "govuk-label",
+          for: "confirmcrn"
+        },
+        classes: "govuk-input govuk-input--width-10",
+        id: "confirmcrn",
+        name: "confirmcrn",
+        value: confirmcrn,
```

Comment:
Where is this value 'confirmcrn' coming from? I think it should be crn instead as it is being set here

<img width="605" alt="image" src="https://user-images.githubusercontent.com/118533775/203567684-dfc7e877-fe37-42b8-af7b-86cb261cb770.png">